vm/interpreter/src/gas_tracker/gas_charge.rs

```diff
@@ -22,4 +24,18 @@ impl GasCharge {
     pub fn total(&self) -> i64 {
         self.compute_gas + self.storage_gas
     }
+
+    pub fn for_charge(&self) -> (&'static str, Gas) {
+        (self.name, Gas::new(self.total()))
+    }
+}
+
+impl From<GasCharge> for fvm::gas::GasCharge<'_> {
+    fn from(charge: GasCharge) -> Self {
+        Self {
+            name: charge.name,
+            compute_gas: Gas::new(charge.compute_gas),
+            storage_gas: Gas::new(charge.storage_gas),
```

Comment:
`fvm::gas::GasCharge` is not exactly the same as our `GasCharge`. You sure you should use `new` rather than `from_milligas`?

Reply:
Changed it

Reply:
Hmm, comparing FVM source code to ours, I don't think we have a concept of milligas. I think our units are just gas.

Reply:
Fixed it back